content/references/rippled-api/commandline-usage.md

```diff
@@ -0,0 +1,149 @@
+| `--start`             | Intended for debugging. Start with a new genesis ledger that has all known amendments (except those the server is configured to vote against) enabled. The functionality of those amendments is therefore available starting from the second ledger, rather than going through the full two-week [Amendment Process](amendments.html). |
+| `---valid`            | **DEPRECATED** Intended for debugging. Consider the initial ledger a valid network ledger even before fully syncing with the network. |
+
+## Client Mode Options
+
+```bash
+rippled [OPTIONS] -- {COMMAND} {COMMAND_PARAMETERS}
+```
+
+In client mode, the `rippled` executable acts as a client to another `rippled` service. (The service may be the same executable running in a separate process locally, or it could be a `rippled` server on another server.)
+
+To run in client mode, provide the [commandline syntax](request-formatting.html#commandline-format) for one of the [`rippled` API](rippled-api.html) methods.
+
+In addition to the individual command syntax, client mode accepts the [Generic Options](#generic-options) and the following options:
+
+| Option                  | Description                                        |
+|:------------------------|:---------------------------------------------------|
+| `--rpc`                 | Explicitly specify that the server should run in client mode. Not required. |
+| `--rpc_ip {IP_ADDRESS}` | Connect to the `rippled` server at the specified IP Address, optionally including a port number. |
+| `--rpc_port {PORT}`     | **DEPRECATED** Connect to the `rippled` server on the specified port. Specify the port alongside the IP address using `--rpc_ip` instead. |
+
+**Tip:** Some arguments accept negative numbers as values. To ensure that arguments to API commands are not interpreted as options instead, pass the `--` argument before the command name.
+
+Example usage (get account transaction history from the earliest available to latest available ledger versions):
+
+```bash
+rippled -- account_tx r9cZA1mLK5R5Am25ArfXFmqgNwjZgnfk59 -1 -1
+```
+
+
+## Unit Tests
+
+```bash
+rippled --unittest [OPTIONS]
+rippled -u [OPTIONS]
+```
+
+Unit testing runs tests built into the `rippled` source code to confirm that the executable performs as expected. This includes things like testing built-in data types and transaction processing routines.
+
+If unit testing reports a failure, that generally indicates one of the following:
+
+- A problem occurred when compiling `rippled` and it is not functioning as intended
+- The source code for `rippled` contains a bug
+- A unit test has a bug or has not been updated to account for new behavior
+
+While running unit tests, you can also provide the following options:
```

Comment:
misses the `--unittest-arg arg`, `--unittest-log`, `--unittest-jobs arg` and the upcoming `--unittest-ipv4` (or `--unittest-ipv6`) flags.

Reply:
Added a task to the backlog to add the IPv4/6 flags when 1.1.0 is closer to ready.  The other ones are added in 3334e3c